Cesium3DTilesSelection/src/TilesetJsonLoader.cpp

```diff
@@ -180,6 +187,72 @@ std::optional<BoundingVolume> getBoundingVolumeProperty(
           maximumHeight,
           ellipsoid);
     }
+
+    const auto cylinderIt = extensionsIt->value.FindMember(
```

Comment:
The code above sets a poor example (probably because it was written before this was possible), but you can use the generated readers here. Something like:

```
Extension3dTilesBoundingVolumeCylinderReader reader;
ReadJsonResult<Cesium3DTiles::Extension3dTilesBoundingVolumeCylinder> cylinderExtensionResult = reader.readFromJson(etensionsIt->value);
```

In fact you can read the entire `BoundingVolume` using `BoundingVolumeReader`.